place_tracker/lib/place_details.dart

```diff
@@ -4,17 +4,17 @@
 
 import 'package:flutter/material.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:provider/provider.dart';
 
 import 'place.dart';
+import 'place_tracker_app.dart';
 import 'stub_data.dart';
 
 class PlaceDetails extends StatefulWidget {
   final Place place;
-  final ValueChanged<Place> onChanged;
 
   const PlaceDetails({
     required this.place,
-    required this.onChanged,
```

Comment:
Callbacks for `onChanged` have been removed, instead the map is updated in the `_watchMapConfigurationChanges` method by listening to `AppState` changes.